docker/Makefile

```diff
@@ -1,15 +1,15 @@
 build:
-	docker build -t wait-for-mysql -f ./deploy-scripts/common/wait-for-mysql/Dockerfile ./deploy-scripts/common/wait-for-mysql
-	docker build -t rdl-athenz-server -f ./util/rdl-athenz-server/Dockerfile ../rdl/rdl-gen-athenz-server
-	docker build -t athenz-mvn-base -f ./util/athenz-mvn-base/Dockerfile ../
-	docker build -t athenz-builder -f ./util/athenz-builder/Dockerfile ../
-	docker build -t athenz-zms-server -f ./zms/Dockerfile ../
-	docker build -t athenz-zts-server -f ./zts/Dockerfile ../
-	docker build -t athenz-ui -f ./ui/Dockerfile ../ui
-	docker build -t athenz-zms-db -f ./db/zms/Dockerfile ../servers/zms/schema
-	docker build -t athenz-zts-db -f ./db/zts/Dockerfile ../servers/zts/schema
-	docker build -t athenz-zms-cli -f ./util/zms-cli/Dockerfile ../
-	docker build -t athenz-cli-util -f ./util/Dockerfile ../
+	DOCKER_BUILDKIT=1 docker build -t wait-for-mysql -f ./deploy-scripts/common/wait-for-mysql/Dockerfile ./deploy-scripts/common/wait-for-mysql
```

Comment:
better to use 1 `export DOCKER_BUILDKIT=1` command instead of adding to each docker build command?

Reply:
it is makefile💩
> each line in a Makefile is a separate shell-process

https://stackoverflow.com/questions/6995107/why-exported-variables-in-makefile-is-not-received-by-executable

Reply:
oops... right... sosad